README.md

```diff
@@ -51,10 +51,9 @@ If after your research you still have a question, ask at [#element-ios:matrix.or
 
 Copyright (c) 2014-2017 OpenMarket Ltd  
 Copyright (c) 2017 Vector Creations Ltd  
-Copyright (c) 2017-2021 New Vector Ltd
+Copyright (c) 2017-2024 New Vector Ltd
```

Comment:
I would have thought these could be merged, but IANAL 🤷‍♂️

Reply:
@manuroe do you know if we could just merge Vector Creations Ltd into New Vector Ltd ? or should we keep them separate?

Reply:
We have to keep them separate. This is the same engineers but not the same company and lawyers.